pkg/config/application.go

```diff
@@ -17,7 +17,7 @@ import (
 // Application is a copy of the orderer configuration with the addition of an anchor peers
 // list in the organization definition.
 type Application struct {
-	Organizations []*Organization
+	Organizations []Organization
 	Capabilities  map[string]bool
 	Policies      map[string]*Policy
```

Comment:
Pointer or value?

Reply:
it is easier to use it as pointers, as for now many tests are modifying policy values, which using pointer make it a bit easier. We can consider changing it to value down the line.